src/main/java/com/hubspot/smtp/client/SmtpSessionConfig.java

```diff
@@ -18,15 +18,20 @@ public Duration getReadTimeout() {
   }
 
   @Default
+  public Duration getKeepAliveTimeout() {
```

Comment:
should this be optional?
If it's specified, should we validate (using `@Check`) that the seconds are not zero so it's activated?